### ISSUE TITLE
Consecutive `var` declarations can be cuddled

### ISSUE THREAD

**Issue by olibre** [bombsimon/wsl#91]
Simplified version of my code:

```go
for result.Next() {
    // some
    // code

    var a string
    var b interface{} // declarations should never be cuddled (wsl)

    for key, val := range result.Values() {
        [...]
```

I have found two options to respect the rule **declarations should never be cuddled** in that case.

1. Add a blank line between the two `var` statements:

    ```go
    for result.Next() {
        // some
        // code

	    var a string

	    var b interface{}

   	    for key, val := range result.Values() {
	        [...]
    ```

2. Group together the `var` statements:

    ```go
    for result.Next() {
        // some
        // code

	    var (
            a string
	        b interface{}
        )

   	    for key, val := range result.Values() {
	        [...]
    ```

But, in my opinion, the original code seems readable.
I suggest that consecutive `var` declarations can be cuddled.
@bombsimon What is your feeling?

**Comment by bombsimon** [comment]
This is actually already implemented in #83 (originally discussed in #82) and merged in #87. #83 is still open but with the tag `golangci-lint-only` whicih means the feature is implemented but not merged upstream to `golangci-lint`.

I guess I could try to merge it into upstream and close the issue, there are actually [two more](https://github.com/bombsimon/wsl/issues?q=is%3Aissue+is%3Aopen+label%3Agolangci-only) issues with this tag.

**Comment by bombsimon** [comment]
My bad, #83 was to cuddle `var` declarations and assignments. Cuddling `var` declarations with `var` declarations has always been supported. It uses the `--allow-cuddle-declarations` flag in `wsl` and is configureable with `allow-cuddle-declarations` in the `.golangci.yml` configuration file.
